zebra/zebra_rib.c

```diff
@@ -466,8 +481,12 @@ static int nexthop_active(afi_t afi, struct route_entry *re,
 	}
 	/* Lookup table.  */
 	table = zebra_vrf_table(afi, SAFI_UNICAST, nexthop->vrf_id);
-	if (!table)
+	if (!table) {
+		if (IS_ZEBRA_DEBUG_RIB_DETAILED)
+			zlog_debug("\t%s: Table not found",
```

Comment:
Adding vrf_id to this debug may be more useful. Tells the user that table not found for a specific vrf-id.

Reply:
I actually did think about this and didn't put this in because a rib detailed situation, the function immediately before this calls a dump of nexthops and all information about the route. This is also why I did the \t to set the information out a bit more.